tests/rptest/tests/cloud_storage_cache_test.py

```diff
@@ -24,3 +26,66 @@ def test_admin_apis(self):
 
             assert "Cloud Storage Cache is not available. Is cloud storage enabled?" == excinfo.value.response.json(
             )['message']
+
+
+class CloudStorageCacheAdminApisTest(RedpandaTest):
+    """
+    Test the Cloud Storage Cache Admin APIs when tiered storage is configured.
+    """
+    def __init__(self, test_context):
+        super().__init__(test_context,
+                         num_brokers=1,
+                         si_settings=SISettings(test_context))
+        self.admin = Admin(self.redpanda)
+
+    def setUp(self):
+        pass
+
+    @cluster(num_nodes=1,
+             log_allow_list=["Free space information is not available"])
+    def test_admin_apis(self):
+        for node in self.redpanda.nodes:
+            node.account.ssh(
+                f"mkdir -p {self.redpanda.cache_dir} ; "
+                "for n in `seq 1 100`; do "
+                f"dd if=/dev/urandom bs=4k count=1 of={self.redpanda.cache_dir}/garbage_$n.bin ; done",
+            )
+
+            self.redpanda.start(clean_nodes=False)
+
+            # Assert initial conditions.
+            usage = self.admin.get_local_storage_usage(node)
+            assert usage['cloud_storage_cache_objects'] == 100, usage
```

Comment:
nit: reduce magic number usage in `dd` command above and here by saving them in variables?

```
num_objects = 100
object_size = 4096
```